### DISCUSSION TITLE
git cloning issue

### DISCUSSION THREAD

**Discussion by Baji8199** [learndevopsonline/learndevopsonline#1047] [Q&A]
Hi @r-devops sir,

When ever I am trying to clone the repository to the local system, facing some issues. Attached are the screenshots, can you guide me to resolve it. Thanks

![Screenshot (6)](https://github.com/learndevopsonline/learndevopsonline/assets/118079306/41eba1d2-09a5-479e-8372-7acb8be157b9)

![Screenshot (7)](https://github.com/learndevopsonline/learndevopsonline/assets/118079306/6200e027-7ff1-4b92-9367-169b9a43df83)

**Comment by learninguser** [comment]
Hi, Can you please check if git is installed on your laptop or not? You can check by opening a command prompt and type `git` in it. If you something like this: 
```
usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]
```
It indicates that git is installed on your laptop. If you don't see something like this, it means the git client is not added to the system path or it is not installed on your laptop.

To install GIT SCM client on your laptop, you can download it from this url: https://github.com/git-for-windows/git/releases/download/v2.41.0.windows.1/Git-2.41.0-64-bit.exe

Thank you.
Kind regards
Pavan Kumar Akula.

**Reply by Baji8199** [reply]
Yes, git is installed and working properly. 

Thanks
